src/plugins/vis_type_timeseries/server/lib/vis_data/series/handle_response_body.ts

```diff
@@ -6,16 +6,24 @@
  * Side Public License, v 1.
  */
 
-import { buildProcessorFunction } from '../build_processor_function';
-import { processors } from '../response_processors/series';
 import { get } from 'lodash';
 import { i18n } from '@kbn/i18n';
-import { createFieldsFetcher } from './../helpers/fields_fetcher';
+import { PanelSchema } from '../../../../common/types';
+import { buildProcessorFunction } from '../build_processor_function';
+// @ts-expect-error
+import { processors } from '../response_processors/series';
+import { createFieldsFetcher, FieldsFetcherServices } from './../helpers/fields_fetcher';
+import { VisTypeTimeseriesVisDataRequest } from '../../../types';
 
-export function handleResponseBody(panel, req, searchStrategy, capabilities) {
-  return async (resp) => {
+export function handleResponseBody(
+  panel: PanelSchema,
+  req: VisTypeTimeseriesVisDataRequest,
+  services: FieldsFetcherServices
+) {
+  return async (resp: any) => {
     if (resp.error) {
       const err = new Error(resp.error.type);
+      // @ts-expect-error
```

Comment:
let's try to refactor it instead of adding `@ts-expect-error` 
From my point of view that error handling doesn't have a lot of sense  espessially if finally we return users only errors related to `KQLSyntaxError`

i think we can replace it to `throw JSON.stringify(resp);`

Reply:
ok, done